docs/dotnet/dotnet-aspnetcore-metrics.md

```diff
@@ -21,6 +21,19 @@ This article defines semantic conventions for ASP.NET Core metrics.
   - [Metric: `aspnetcore.rate_limiting.queued_requests`](#metric-aspnetcorerate_limitingqueued_requests)
   - [Metric: `aspnetcore.rate_limiting.request.time_in_queue`](#metric-aspnetcorerate_limitingrequesttime_in_queue)
   - [Metric: `aspnetcore.rate_limiting.requests`](#metric-aspnetcorerate_limitingrequests)
+- [Identity](#identity)
```

Comment:
Do you have a plan to mark ASP.NET Core versions supporting given metrics, or it should be done on implementer side?
I suppose in this case it will be .NET10+.

@JamesNK, any plans to extend instrumentation package in the contrib with new activity source?

Reply:
Sorry, let me rephrase it.
Do you think that this document should contain information about ASP.NET Core version supporting given set of versions or it should be done only on .NET documentation side.
As I understand, Identity _space_ will be supported only by ASP.NET Core 10+.

Reply:
Each metric has a note that says:

> Meter name: `Microsoft.AspNetCore.Identity`; Added in: ASP.NET Core 10.0